spec/models/code_ocean/file_spec.rb

```diff
@@ -3,15 +3,21 @@
 require 'rails_helper'
 
 RSpec.describe CodeOcean::File do
-  let(:file) { described_class.create.tap {|file| file.update(content: nil, hidden: nil, read_only: nil) } }
+  let(:file) do
+    described_class.new.tap do |file|
```

Comment:
I want to make clear that the `create` does not create a record here. Therefore, I use assignment and call validate.